internal/action/actions.go

```diff
@@ -856,30 +856,6 @@ func (h *BufPane) FindLiteral() bool {
 	return h.find(false)
 }
 
-// Search searches for a given string/regex in the buffer and selects the next
-// match if a match is found
-// This function affects lastSearch and lastSearchRegex (saved searches) for
-// use with FindNext and FindPrevious
-func (h *BufPane) Search(str string, useRegex bool, searchDown bool) error {
```

Comment:
Why has this function been removed?

Reply:
Just cleaning...
It is **not** used and is confusing (with other findxxx methods)

Reply:
We should keep this function because it may be used by a plugin.